transit_odp/avl/client/cavl.py

```diff
@@ -166,3 +168,100 @@ def validate_feed(
                 created=timezone.now(),
                 version=siri_version,
             )
+
+
+class CAVLSubscriptionService(ICAVLSubscriptionService):
+    AVL_CONSUMER_URL = settings.AVL_CONSUMER_API_BASE_URL
+
+    @staticmethod
+    def _get_error_response(exception: RequestException) -> str:
+        return (
+            exception.response.json()
+            if hasattr(exception.response, "json")
+            else "(empty)"
+        )
+
+    def subscribe(
+        self,
+        api_key: str,
+        name: str,
+        url: str,
+        update_interval: int,
+        subscription_id: str,
+        dataset_ids: str,
+        bounding_box: str,
+        operator_ref: str,
+        vehicle_ref: str,
+        line_ref: str,
+        producer_ref: str,
+        origin_ref: str,
+        destination_ref: str,
+    ) -> None:
+        api_url = f"{self.AVL_CONSUMER_URL}/v1/siri-vm/subscriptions?name={name}&subscriptionId={dataset_ids}"
```

Comment:
```suggestion
        api_url = f"{self.AVL_CONSUMER_URL}/siri-vm/subscriptions?name={name}&subscriptionId={dataset_ids}"
```

We should remove v1 here to match the rest of the URLs